src/bin/tikv-ctl.rs

```diff
@@ -16,6 +16,7 @@
 #![cfg_attr(not(feature = "dev"), allow(unknown_lints))]
 #![allow(needless_pass_by_value)]
 
+#[macro_use]
```

Comment:
Does it necessary?

Reply:
Yes, it is added in another PR, but that is not picked to release-2.0